### DISCUSSION TITLE
Handle beanie.exceptions.RevisionIdWasChanged in beanie 1.19.2

### DISCUSSION THREAD

**Discussion by NiCkWKT** [BeanieODM/beanie#593] [Question]
The exception occurs when I update beanie from 1.10.9 to 1.19.2

At first, since Collection inner class is deprecated, so I simply change the class name to Settings.

```python
@user_router.post('/create', summary="Create new user")
async def create_user(data: UserAuth):
    try:
        return await UserService.create_user(data)
    except pymongo.errors.DuplicateKeyError:
        raise HTTPException(
            status_code=status.HTTP_400_BAD_REQUEST,
            detail="User with this email or username already exist"
        )
```

When I am trying to catch the above exception, I got `During handling of the above exception, another exception occurred:`, `beanie.exceptions.RevisionIdWasChanged`

```python
class UserService:
    @staticmethod
    async def create_user(user: UserAuth):
        user_in = User(
            username=user.username,
            email=user.email,
            hashed_password=get_password(user.password)
        )
        await user_in.save()
        return user_in
```
The above `User` class is inherited from `Document`

This works with no error in 1.10.8. How should I avoid or handle this exception in 1.19.2

Many thanks!

**Comment by paul1tsch** [comment]
Consider using `user_in.insert()` instead of `user_in.save()`.
